corehq/apps/translations/app_translations/upload_form.py

```diff
@@ -193,8 +193,8 @@ def _add_or_remove_translations(self, lang, row):
 
             if trans_type == 'default':
                 # plaintext/Markdown
-                markdown_allowed = not self.markdown_vetoes[label_id] or self.markdowns[label_id]
-                if self._looks_like_markdown(new_translation) and markdown_allowed:
+                is_markdown = self._looks_like_markdown(new_translation) and not self.markdown_vetoes[label_id]
+                if is_markdown or self.markdowns[label_id]:
```

Comment:
Can i request for a better naming here and a better documentation?
this is not straight forward/easy to understand coz it is receiving lot of context bound on the object and needs quite some lookup. Something like this but correct
```
was_markdown = self.markdowns[label_id]
is_markdown = self._looks_like_markdown(new_translation)
has_markdown_vetoed = self.markdown_vetoes[label_id] # the markdown that takes preference over this one or whatever this means as a comment

update_markdown = is_markdown and not has_markdown_vetoed
if update_markdown or was_markdown:
    ....
```

Reply:
7c2eebe641a - I did a long comment because this is tricky to explain without a lot of words.

Reply:
thanks. Would have liked the rename.. but this is fine too.

nit: `markdown_vetoes` to `markdown_to_reject` / `ignore_markdowns`/ `markdown_to_be_rejected`

Reply:
Leaving this alone. There's other code that uses "veto" and I don't want to change all of it.